_sass/minimal-mistakes.scss

```diff
@@ -104,3 +104,12 @@
     font-weight: bold;
   }
 }
+
+.clients{
```

Comment:
Uma melhoria aqui seria extrair essa classe pra um arquivo específico de estilos do bloco 5 e incluir ela como um partial no minimal-mistakes.scss. Se tu achar que é melhor, lá no meu PR tem esse jeito

Reply:
@leo-holanda como tu fez para associar o arquivo com este nome _sixth_block.scss ao @import "minimal-mistakes/outorga-page/sixth_block"; porque aqui o sixth_block não tem o traço no início.

Reply:
Rapaz, acho que isso vem do SASS. Os arquivos de partials sempre começam com underline, mas não precisa citar eles no import.